README.md

```diff
@@ -1,38 +1,106 @@
 [![Automatic version updates](https://github.com/ZOSOpenTools/v8port/actions/workflows/bump.yml/badge.svg)](https://github.com/ZOSOpenTools/v8port/actions/workflows/bump.yml)
 
-v8
+# v8
 
 A port of Google V8 JavaScript engine to z/OS Open Tools project
 
-# Applying the patches
+## Workflow 
+
+### Applying the patches
 Since the [cipd executable](https://chromium.googlesource.com/infra/luci/luci-go/+/master/cipd/) is not yet ported on z/OS, the following steps are
 currently used to update V8 and its dependencies, and apply the z/OS port, after
 the initial [fetch v8](https://v8.dev/docs/source-code) (also included below with a comment).
 
+This workflow requires two platforms.  One is USS which is the target platform.  The second is a generic linux
+AMD64 platform to get files normally acquired via `cipd`
+
+
+#### Linux AMD64 plaform setup
+
+This is the workflow on the non USS platform for obtaining files via `cipd`.  Do
+this once.
+
 ```
-# Do this only once:
-mkdir ~/v8base
-cd v8base
-fetch v8
-cd v8
+$ cd $HOME
+$ git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+$ export PATH=~/depot_tools:$PATH
+$ fetch v8
+$ cd v8
+$ git checkout main
+$ git reset fce38915e4c7c73c0e7f77bb13e2a78514968d35 --hard
+$ tar -cvf x.tar test tools third_party
+$ scp x.tar user@uss_host:~/zopen/.
```

Comment:
I suggest using `sftp` instead as mentioned in the original instructions.

Reply:
Hmm, this is interesting.  I used scp and sftp to copy the tar file to USS..  I did the md5sum on USS and linux.
Although the file sizes are equivalent, there is a problem with scp.  I have used scp a million times on this box and this is the first time I have seen this error before.  Tip of the hat to you.

Yes, so sftp is the method to use, but its odd.  The files have the same size, the tagging is identicial, and the file command reports them as tar files.  Very odd.  Any idea why one works and the other doesn't?

### USS

First is the one using scp.  The other was done using sftp as you suggested.

```
$ ls -l x.tar*
-rw-r--r--   1 JD895801 TTOODGRP 2505728000 Feb  1 16:15 x.tar
-rw-r--r--   1 JD895801 TTOODGRP 2505728000 Feb  1 10:40 x.tar.first
$ md5sum x.tar x.tar.first
2994f564d99c07adc7ed91ca83cb86ed *x.tar
fbc7bdbcf113a4ab639b6cabb9a95705 *x.tar.first
$ chtag -p x.tar*
- untagged    T=off x.tar
- untagged    T=off x.tar.first
JD895801@USILCA31 zopen
$ file x.tar*
x.tar:	USTAR format tar archive
x.tar.first:	USTAR format tar archive
```

### Linux

```
davis@jd895801-2024-02-1-9-26-24-sandbox01:~/v8$ md5sum x.tar
2994f564d99c07adc7ed91ca83cb86ed  x.tar
davis@jd895801-2024-02-1-9-26-24-sandbox01:~/v8$ ls -l x.tar
-rw-r--r-- 1 davis davis 2505728000 Feb  1 15:29 x.tar
```